include/wpe/renderer-backend-egl.h

```diff
@@ -36,6 +36,7 @@
 #endif
 
 #include <EGL/eglplatform.h>
+#include <EGL/egl.h>
```

Comment:
I'm not in favor of this include. It prevents from building libwpe with toolchains that don't provide EGL headers (but can build with a custom or Khronos-provided generic eglplatform.h header).[1]

I'd prefer if EGLenum usage is replaced with uint32_t. Conversion between the two shouldn't be a problem because they're both 32-bit unsigned types. uint32_t is thus probably fine, unless any future EGL specification ends up expanding EGLenum into a larger type.

[1] WebKit works around that by using libepoxy, which similarly only requires the eglplatform.h header to build its EGL capabilities. What in the current WPE stack doesn't comply with this approach is the GStreamer's GL functionality.